roadmap/implementers-guide/src/runtime/disputes2.md

```diff
@@ -0,0 +1,54 @@
+# Disputes
+
+## Motivation
+
+If the transaction can be verified the defined verification code, it is good. If not, there is a dispute
+about a block, which made it through the backing stage, but was then during secondary checking found
+to be invalid.
```

Comment:
It's technically possible two backers disagree too.  We should not waste too much time on this special case because a backing voter could create the dispute exactly like an approval checker can, but then availability should run concurrently with the dispute, and the disputing backer should continue handing out pieces, just like if they were a backing vote.  

An incorrect disputer need not be slashed 100% but they should be slashed enough that censoring a parachain by disputing blocks you dislike becomes prohibitively expensive, and correlated slashing rule apply to drive this upwards towards 100%.